src/ui/screenshot_menu.ts

```diff
@@ -326,16 +401,28 @@ export class ScreenshotDialog extends Overlay {
     this.statisticsTable.classList.add(
       "neuroglancer-screenshot-statistics-table",
     );
-    this.statisticsTable.title = "Screenshot statistics";
+    this.statisticsTable.title = "Screenshot progress";
```

Comment:
I think we can remove the occurences of setting .title in this file. Now that the UI is better designed, this info on hover is not really needed